cmd/internal/cli/pull.go

```diff
@@ -60,9 +59,9 @@ var (
 var pullArchFlag = cmdline.Flag{
 	ID:           "pullArchFlag",
 	Value:        &pullArch,
-	DefaultValue: runtime.GOARCH,
+	DefaultValue: "",
 	Name:         "arch",
-	Usage:        "architecture to pull from library",
+	Usage:        "architecture to pull, should be one of ('amd64', 'arm32v5', 'arm32v6', 'arm32v7', 'arm64v8', '386', 'ppc64le', 's390x')",
```

Comment:
This is shared with sylabs library, I think they are following naming based on GOARCH, so this usage message could be confusing depending on pulling from `docker://` or `library://`, maybe put the list for both in command long description here https://github.com/apptainer/apptainer/blob/e21533c26da455921b365b864f55dd78fd37bfc0/docs/content.go#L617